widget/ui/src/components/Toast/Toast.Provider.tsx

```diff
@@ -62,14 +53,19 @@ export const ToastProvider = (props: PropsWithChildren<ProviderPropTypes>) => {
       {children}
       {container &&
         createPortal(
-          <ToastContainer
-            style={containerStyle}
-            horizontal={anchorOrigin.horizontal}
-            vertical={anchorOrigin.vertical}>
-            {toasts.map((toast) => (
-              <Toast {...toast} key={toast.id} />
-            ))}
-          </ToastContainer>,
+          <>
+            {toasts.map(
+              ({ containerStyle, horizontal, vertical, ...toast }) => (
+                <ToastContainer
+                  key={toast.id}
+                  style={containerStyle}
+                  horizontal={horizontal || 'center'}
+                  vertical={vertical || 'top'}>
+                  <Toast {...toast} key={toast.id} />
+                </ToastContainer>
+              )
+            )}
```

Comment:
If we have two parallel toast with different vertical props, does it work well?

Reply:
I changed the structure to handle these situations.